app/assets/javascripts/mapknitter/Map.js

```diff
@@ -88,23 +90,30 @@ MapKnitter.Map = MapKnitter.Class.extend({
 
           });
 
-          var corners = [ 
-                new L.latLng(warpable.nodes[0].lat,
-                             warpable.nodes[0].lon),
-                new L.latLng(warpable.nodes[1].lat,
-                             warpable.nodes[1].lon),
-                new L.latLng(warpable.nodes[3].lat,
-                             warpable.nodes[3].lon),
-                new L.latLng(warpable.nodes[2].lat,
-                             warpable.nodes[2].lon)
+          var corners = [
+            L.latLng(warpable.nodes[0].lat,
+              warpable.nodes[0].lon),
+            L.latLng(warpable.nodes[1].lat,
+              warpable.nodes[1].lon),
+            L.latLng(warpable.nodes[3].lat,
+              warpable.nodes[3].lon),
+            L.latLng(warpable.nodes[2].lat,
+              warpable.nodes[2].lon)
           ];
 
-          var img = new L.DistortableImageOverlay(
+          var img = L.distortableImageOverlay(
             warpable.srcmedium,
-            { 
+            {
               keymapper: false,
+              actions: mapknitter.imgActionArray(),
               corners: corners,
               mode: 'lock'
+            }).addTo(map);
+
+          var exportA = mapknitter.customExportAction();
```

Comment:
OK, i'm moving where we instantiate the collection because I think we're doing it repeatedly (for each image) currently... i'm moving it up.